internal/system/agent/const.go

```diff
@@ -0,0 +1,69 @@
+/*******************************************************************************
+ * Copyright 2017 Dell Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License
+ * is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing permissions and limitations under
+ * the License.
+ *******************************************************************************/
+package agent
+
+type ConfigurationStruct struct {
+	ConsulProfilesActive       string
```

Comment:
Does your service really need all of this for Configuration? Only include what you need. You can add back later.